roles/variables/defaults/main.yml

```diff
@@ -1509,7 +1509,9 @@ oauth_expected_audience: none
 oauth_super_user: "{{oauth_client_id}}"
 
 ### SSL certificate (full path of file on control node) of IDP Domain. Optional, needed when IDP server has TLS enabled with custom certificate
-oauth_idp_pem_path: ""
+oauth_idp_cert_path: ""
+
+idp_self_signed: "{{'true' if oauth_idp_cert_path != '' else 'false'}}"
```

Comment:
is it necessary that the idp cert is self signed when customer provides a cert. I thought it can also be signed by third party

Reply:
if it's signed by third party, it is trusted and we don't need to import it